bigbluebutton-html5/imports/ui/components/shortcut-help/component.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from 'react';
+import { defineMessages, injectIntl } from 'react-intl';
+import Modal from '/imports/ui/components/modal/simple/component';
+import styles from './styles.scss';
+
+const intlMessages = defineMessages({
+  shortcutTitle: {
+    id: 'app.shortcut-help.shortcutTitle',
+    description: 'title for shortcut help modal',
+  },
+  cancelLabel: {
+    id: 'app.shortcut-help.cancelLabel',
+    description: 'label for dismiss modal button',
+  },
+  cancelDesc: {
+    id: 'app.shortcut-help.cancelDesc',
+    description: 'description for dismiss modal button',
+  },
+  firstColTitle: {
+    id: 'app.shortcut-help.firstColTitle',
+    description: 'heading for shortcuts column',
+  },
+  secondColTitle: {
+    id: 'app.shortcut-help.secondColTitle',
+    description: 'heading for shortcut function column',
+  },
+  shortcutFunc001: {
+    id: 'app.shortcut-help.shortcutFunc001',
+    description: 'Toggle fullscreen shortcut description',
+  },
+  shortcutFunc002: {
+    id: 'app.shortcut-help.shortcutFunc002',
+    description: 'Toggle userlist shortcut description',
+  },
+  shortcutFunc003: {
+    id: 'app.shortcut-help.shortcutFunc003',
+    description: 'Mute/Unmute shortcut description',
+  },
+  shortcutFunc004: {
+    id: 'app.shortcut-help.shortcutFunc004',
+    description: 'Logout shortcut description',
+  },
+  shortcutFunc005: {
+    id: 'app.shortcut-help.shortcutFunc005',
+    description: 'Open about shortcut description',
+  },
+  shortcutFunc006: {
+    id: 'app.shortcut-help.shortcutFunc006',
+    description: 'Open settings modal shortcut description',
+  },
+  shortcutFunc007: {
+    id: 'app.shortcut-help.shortcutFunc007',
+    description: 'Previous slide shortcut description',
+  },
+  shortcutFunc008: {
+    id: 'app.shortcut-help.shortcutFunc008',
+    description: 'Next slide shortcut description',
+  },
+  shortcutFunc009: {
+    id: 'app.shortcut-help.shortcutFunc009',
+    description: 'Upload presentation shortcut description',
+  },
+  shortcutFunc010: {
+    id: 'app.shortcut-help.shortcutFunc010',
+    description: 'Open shortcut help modal description',
+  },
+});
+
+class ShortcutHelpComponent extends Component {
+  render() {
+    const { intl } = this.props;
+
+    const shortcuts = [
+      { keys: 'Control + Alt + 0', function: intl.formatMessage(intlMessages.shortcutFunc001) },
```

Comment:
Since you are using an number of functions, why not have the key combination in a separate file (maybe config)? Then you can reuse on both the component and here.

Reply:
Can you use some other name than `function`? Kinda confusing a function key storing a string/message.

I think `description` would be a good name, since the message describe what the shortcut does.